markdown/design-to-code-workshop/design-to-code-workshop.md

```diff
@@ -0,0 +1,591 @@
+<p>Terms of Use: This dataset is publicly available for anyone to use under the following terms provided by the Dataset Source — [https://data.cityofnewyork.us/](https://data.cityofnewyork.us/) — and is provided "AS IS" without any warranty, express or implied, from Solace. Solace disclaims all liability for any damages, direct or indirect, resulting from the use of the dataset.</p>
+
+## What You'll Need
+Duration: 0:08:00
+
+🛠 This page covers the setup needed to perform this codelab. 🛠 
+
+### AsyncAPI Generator Requirements
+✅ Install instructions available [here](https://github.com/asyncapi/generator#requirements)
+* Node.js v12.16+ 
+* npm v6.13.7+
+
+We'll install the generator itself later 👍
+
+### Spring Cloud Stream Requirements
+✅ Spring Cloud Stream just requires Java and Maven to use 🚀  
+* Java 1.8+ 
+* Maven 3.3+
+	* On mac you can `brew install maven`
+	* Other install instructions [here](https://maven.apache.org/install.html)
+* Your favorite Java IDE 💥
+
+### PubSub+ Event Broker Connection Info
+✅ The credentials below are for a public event feed that we'll use during this codelab.
+* SMF Host: `tcp://mr-d8f4yze27kt.messaging.solace.cloud:55555`
+* Message VPN: `cto-demo-virginia-azure`
+* Username: `public-taxi-user`
+* Password: `iliketaxis`
+
+✅ Note that this client-username has permissions to subscribe to `taxi/>` and `test/taxi/>` and permissinos to publish to `test/taxi/>`
+
+### Prepare PubSub+ Event Portal
+
+#### Sign-up for Solace Cloud
+✅ If you already have a Solace Cloud account just login, otherwise please sign-up for a free Solace Cloud Account using [this link](bit.ly/try-solace-free). Note that no credit card is required. You will receive an email to activate the account and will then be prompted to start the free trail. 
+
+![sc_trial](img/sc_trial.webp)
+
+
+#### Import Application Domain
+✅ Download the Application Domain export file: [EventPortal_Export_NYCModernTaxiCo.json]( https://github.com/Mrc0113/design-to-code-workshop/blob/master/EventPortal_Export_NYCModernTaxiCo.json)
+
+You can download the file via curl or by cloning the git repo
+```bash 
+curl -XGET https://raw.githubusercontent.com/Mrc0113/design-to-code-workshop/master/EventPortal_Export_NYCModernTaxiCo.json -o EventPortal_Export_NYCModernTaxiCo.json
+``` 
+OR
+```bash
+git clone https://github.com/Mrc0113/design-to-code-workshop.git
+```
+
+✅ Inside of your logged into Solace Cloud Account navigate to the Event Portal Designer by clicking "Designer" in the menu on the left. 
+
+![ep_select_designer](img/ep_select_designer.webp)
+
+✅ Then import the previously downloaded Application Domain file by clicking the `Import` button at the top right of the _Designer_ and importing the  file. 
```

Comment:
Isint there a limit to two application domains in the free account? I recall that the free account comes pre populated with two application domains - the rideshare and a default one (unless this changed)